Docker/transforms/shorthand2xccdf.xslt

```diff
@@ -0,0 +1,298 @@
+<?xml version="1.0"?>
```

Comment:
This XSLT needs to be updated to use `shared_shorthand2xccdf.xslt`. See `RHEL/7/transforms/shorthand2xccdf.xslt` for how this is implemented.